docs/user-guide/cluster-configuration.md

```diff
@@ -22,6 +22,56 @@ This guide provides instructions for Day‑2 configuration change use cases, to
 
 ## Use Cases
 
+Day 2 configuration changes are supported for both hardware configuration updates and policy parameter changes. The system supports retry scenarios even after previous configuration attempts have timed out or failed.
+
+### Hardware Configuration Timeouts and Retry
+
+Hardware configuration timeout detection is handled by the Metal3 hardware plugin. When a configuration operation times out or fails, the system supports retry through spec changes.
```

Comment:
This is user documentation, we don;t need the first sentence

Reply:
The first sentence has been removed.